### ISSUE TITLE
[API] Incorrect use of “representation”

### ISSUE THREAD

**Issue by RubenVerborgh** [openknowledgebe/open-education#10]
The API documentation describes subjects as "Subjects are representations of a subject".
This is terminologically incorrect; subject are subjects, not representations; i.e., <http://example.org/subjects/Mathematics> identifies the concept of maths, not a representation of the concept.

A correct (and less tautological) way to describe things would be "A subject is a branch of knowledge studied or taught in an educational institution". Similar for other properties.

**Comment by jhoobergs** [comment]
Indeed, I changed this from something else and now it doesn't really mean anything anymore. I'll fix it.
